finetune/base.py

```diff
@@ -435,6 +441,29 @@ def _define_placeholders(self):
         self.Y = tf.stop_gradient(tf.placeholder(tf.float32, [None, self.target_dim or 1])) # classification targets
         self.do_dropout = tf.placeholder(tf.float32)  # 1 for do dropout and 0 to not do dropout
 
+    def model_language(self, max_length=None, seed_text=''):
```

Comment:
Maybe... generate_text?  Model_language just sounds like a training method to me.